connectors/sources/confluence.py

```diff
@@ -1178,6 +1269,16 @@ async def _consumer(self):
             else:
                 yield item
 
+    def check_api_exceptions_and_raise(self):
+        failed_percentage = (
+            self.confluence_client.api_failed_count
+            / self.confluence_client.api_total_count
+        ) * 100
```

Comment:
```suggestion
        failed_percentage = round((
            self.confluence_client.api_failed_count
            / self.confluence_client.api_total_count
        ) * 100, 2)
```
let's limit the number of digits here so that the error message isn't obscene